roles/kubernetes/master/templates/manifests/kube-controller-manager.manifest.j2

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kube-controller-manager
+  namespace: kube-system
+spec:
+  hostNetwork: true
+  containers:
+  - name: kube-controller-manager
+    image: {{ hyperkube_image.name }}:{{ hyperkube_image.tag }}
+    command:
+    - /hyperkube
+    - controller-manager
+    - --master=http://127.0.0.1:8080
```

Comment:
I don't know why we would want to run the controller-manager or the scheduler on other nodes than the masters. I'll think about it. 
There's no security issues binding the apiserver on localhost.